### ISSUE TITLE
Deploy terraform code in relation to #16235 in production

### ISSUE THREAD

**Issue by chris-kuryak** [CDCgov/prime-reportstream#16524]
# DevSecOps Issue

## Summary
Terraform needs to be updated to include the changes found in #16235: Add Environment variable "JAVA_OPTS" with value ""-Dfile.encoding=UTF-8" to all env terraform files.

Some background from James Gilmore:
#16235 makes the deployed environments match the local one with respect to the default encoding so that any characters after decimal 127 actually display. The issue was that I believe the prior devops teams was manually applying terraform changes somehow but I'm not clear on how. I added the env variable to the staging env through the azure portal for testing but AFAIK it has not been applied to production.

The update to the .tf files was deployed to prod with the 10/17 deployment (#16251), but that doesn't appear to have updated the terraform in prod to include the env variable.

## Environment

- [ ] Local
- [ ] Dev
- [ ] Stage
- [x] Prod

## Priority

- [ ] **Critical** - affecting prod systems
- [x] **Major** - blocking major functionality, deployment, etc
  - This is blocking the ETOR Newborn Screening project from moving to production with their CA pilot, which is slated for Nov 18. 
- [ ] **Minor** - improvements, bug fixes
- [ ] **Nice-to-have** - feature request

### Blocks the following

- List any blockers
  - None known
- Link any Github relevant issues
  - #16235

### Blocked by the following

- Link to any tickets that add context, etc

## Contact

**Team**: 

**Slack Team Channel**:

**Slack Contact (Full Name):**

**Technical Team Lead:**

## Definition of Done

- [ ] Add environment variable "JAVA_OPTS" with value ""-Dfile.encoding=UTF-8" to all env terraform files in production (a previously manual process by DevOps)

## Context Links

**Git Repo:**

**Relevant Code Links:**

**Azure**:

**Site URL**:

**Other**:

## Notes

-

**Comment by chris-kuryak** [comment]
Duplicate of Arnej's ticket #16528 .
Closing this one.
